zh_cn/deco.gittertreppe.json

```diff
@@ -1,20 +0,0 @@
-			"8": [{"name":"futurepack:metal_gitter_block", "meta":0},{"name":"futurepack:color_gitter", "meta":0},{"name":"futurepack:color_gitter", "meta":1},{"name":"futurepack:color_gitter", "meta":2},{"name":"futurepack:color_gitter", "meta":3},{"name":"futurepack:color_gitter", "meta":4},{"name":"futurepack:color_gitter", "meta":5},{"name":"futurepack:color_gitter", "meta":6},{"name":"futurepack:color_gitter", "meta":7},{"name":"futurepack:color_gitter", "meta":8},{"name":"futurepack:color_gitter", "meta":9},{"name":"futurepack:color_gitter", "meta":10},{"name":"futurepack:color_gitter", "meta":11},{"name":"futurepack:color_gitter", "meta":12},{"name":"futurepack:color_gitter", "meta":13},{"name":"futurepack:color_gitter", "meta":14},{"name":"futurepack:color_gitter", "meta":15}],
-			"9": [{"name":"futurepack:metal_gitter_block", "meta":0},{"name":"futurepack:color_gitter", "meta":0},{"name":"futurepack:color_gitter", "meta":1},{"name":"futurepack:color_gitter", "meta":2},{"name":"futurepack:color_gitter", "meta":3},{"name":"futurepack:color_gitter", "meta":4},{"name":"futurepack:color_gitter", "meta":5},{"name":"futurepack:color_gitter", "meta":6},{"name":"futurepack:color_gitter", "meta":7},{"name":"futurepack:color_gitter", "meta":8},{"name":"futurepack:color_gitter", "meta":9},{"name":"futurepack:color_gitter", "meta":10},{"name":"futurepack:color_gitter", "meta":11},{"name":"futurepack:color_gitter", "meta":12},{"name":"futurepack:color_gitter", "meta":13},{"name":"futurepack:color_gitter", "meta":14},{"name":"futurepack:color_gitter", "meta":15}],
-			"out": [{"name":"futurepack:metal_gitter_treppe", "meta":0, "size":4},{"name":"futurepack:color_gitter_stair0", "size":4},{"name":"futurepack:color_gitter_stair1", "size":4},{"name":"futurepack:color_gitter_stair2", "size":4},{"name":"futurepack:color_gitter_stair3", "size":4},{"name":"futurepack:color_gitter_stair4", "size":4},{"name":"futurepack:color_gitter_stair5", "size":4},{"name":"futurepack:color_gitter_stair6", "size":4},{"name":"futurepack:color_gitter_stair7", "size":4},{"name":"futurepack:color_gitter_stair8", "size":4},{"name":"futurepack:color_gitter_stair9", "size":4},{"name":"futurepack:color_gitter_stair10", "size":4},{"name":"futurepack:color_gitter_stair11", "size":4},{"name":"futurepack:color_gitter_stair12", "size":4},{"name":"futurepack:color_gitter_stair13", "size":4},{"name":"futurepack:color_gitter_stair14", "size":4},{"name":"futurepack:color_gitter_stair15", "size":4}]
-		}
-	},
-	"信息：可使用喷枪进行染色"
-]
```

Comment:
why was this file deleted?

Reply:
They have no practical effect.
There is no corresponding file in folder [en_us](https://github.com/Wugand/FuturePack-Language/tree/master/en_us).
This recipe is repeated with [deco.gitter.json](https://github.com/Wugand/FuturePack-Language/blob/master/zh_cn/deco.gitter.json#L7)(`	{"link":"futurepack:lang/part/cr_deco.gittertreppe.json"},`)。